RNPComponents/RNPComponents.md

```diff
@@ -0,0 +1,47 @@
+# React Native Paper Component Templates
+
+This Document contains a set of components Templates around various [React Native Paper](https://callstack.github.io/react-native-paper/index.html) components.
+
+By default, the theming mechanism provided by RNP is very minimal and does not allow us to style components precisely the way we want for Brightlayer UI applications. To circumvent this issue, we create a template that you can use bring them in alignment with Brightlayer UI applications
+
+These components templates simplify this process so that the end user do not create the component designs to bring them in alignment with Brightlayer UI applications.
+
+## Component Template
+
+- ###  ProgressBar
+
+In Progress Bar we have updated the Background of Progress bar to SurfaceContainerHighest to match the design
```

Comment:
For Progress Bar we have updated the backgroundColor of the Progress bar to use SurfaceContainerHighest to match the designs for both light and dark themes from our Brightlayer-UI React Native Theme package. The Progress Bar components accepts all the same props from React Native Paper component.

Reply:
That's a lot of content for a relatively self-explanatory thing by the code, IMO...

```
## Progress Bar

Recommended style overrides:
* background color of the track: surfaceContainerHighest

<img ... />

code...
```

Reply:
I like that you included images. Easy to navigate.